pyfar/classes/audio.py

```diff
@@ -781,6 +780,12 @@ def _return_item(self, data):
                       fft_norm=self.fft_norm, comment=self.comment)
         return item
 
+    def _encode(self):
+        """Return dictionary for the encoding."""
+        self.domain = "time"
+        class_dict = self.copy().__dict__
```

Comment:
I don't know if it is crucial, but it might make sense to change the domain of the copy instead? Something like
```suggestion
        selfcopy = self.copy()
        selfcopy.domain = "time"
        class_dict = selfcopy.__dict__
```